src/init/init.ts

```diff
@@ -43,14 +43,17 @@ const templateDescriptions: {[name: string]: string} = {
   'shop': 'The "Shop" Progressive Web App demo',
 };
 
+// TODO: Add Shop "^2.0.0" generator once Polymer 2.0 Shop template is released.
 let shopGenerator = createGithubGenerator({
   owner: 'Polymer',
   repo: 'shop',
+  semverRange: '^1.0.0',
 });
-
+// TODO: Add Shop "^3.0.0" generator once Polymer 2.0 PSK template is released.
```

Comment:
Yea, happy to talk more about this when the time comes. I don't know if we could ever do a truely automated solution, since major version bumps don't necessarily map 1:1 with Polymer version bumps. So if we showed the last two majors, Polymer 1.0 would immediately get bumped off the list.

I hope that we can make the 1.0 -> 2.0 transition time (especially for someone creating a new project) as short as possible. Once feasible, not encouraging people to create 1.0 projects with `init` would support that goal.